e3sm_diags/driver/utils/diurnal_cycle_xr.py

```diff
@@ -70,8 +70,8 @@ def composite_diurnal_cycle(
         nlat = 1
         nlon = 1
 
-        lat = [lat]  # type: ignore
-        lon = [lon]  # type: ignore
+        lat = [ds.lat.values]
+        lon = [ds.lon.values]
```

Comment:
Same comment from [above](https://github.com/E3SM-Project/e3sm_diags/pull/842/files#r1750985331) about these optimizations might not being needed anymore.